pkg/node/client.go

```diff
@@ -0,0 +1,165 @@
+package node
+
+import (
+	"context"
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog"
+
+	"github.com/kubernetes-sigs/container-object-storage-interface-api/apis/objectstorage.k8s.io/v1alpha1"
+	cs "github.com/kubernetes-sigs/container-object-storage-interface-api/clientset/typed/objectstorage.k8s.io/v1alpha1"
+)
+
+type NodeClient struct {
+	cosiClient *cs.ObjectstorageV1alpha1Client
+	kubeClient kubernetes.Interface
+}
+
+func NewClient() (*NodeClient, error) {
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		return nil, err
```

Comment:
If err is expected to be fatal then perhaps a `panic` call should be here rather than at the caller?

Reply:
Fixed!